samples/csharp_dotnetcore/07.using-adaptive-cards/AdaptiveCardsBot.csproj

```diff
@@ -13,7 +13,7 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="AdaptiveCards" Version="1.0.3" />
+    <PackageReference Include="AdaptiveCards" Version="1.1.2" />
```

Comment:
We need to reach out to William on the WebChat team to make sure we're not introducing unexpected behavior by picking up 1.1.2 of AC.  There has been breaking changes made that didn't adhere to semver rules.  we may be OK here, but I'd like William to give us the "ok"

Reply:
I sent an email to Will asking for any know issues. For now, I reverted the change to 1.0.3 so we don't hold this PR, we can upgrade in another PR once he confirms it is OK to bump up the version.